djstripe/templates/djstripe/subscribe_form.html

```diff
@@ -51,16 +47,16 @@
                     <img src="{% static image %}" class="img-thumbnail" />
                   {% endwith %}
                   <h3>{{ plan.name }}</h3>
-                  <p>{{ plan.description }}</p>
                 </button>
 
               {% if not customer.current_subscription or customer.current_subscription.status == CurrentSubscription.STATUS_CANCELLED %}
-                <!-- do nothing -->
-              {% elif customer.current_subscription.plan == plan.plan %}
+              {# do nothing #}
+              {% elif customer.current_subscription.plan == plan.interval and customer.current_subscription.amount == plan.amount %}
+              {# check if user plan owner #}
                 <h4>Your Current Plan</h4>
-              {% elif customer.current_subscription.amount < plan.price|djdiv:100 %}
+              {% elif customer.current_subscription.amount < plan.amount %}
```

Comment:
The Plan model should still keep a price in Decimal format. We can't call it amount because that would cause confusion.